### DISCUSSION TITLE
PMOD Adapter board

### DISCUSSION THREAD

**Discussion by kasbert** [xjtuecho/EBAZ4205#41] [Ideas]
I have been drawing an adapter board. The idea is to expose most pins as PMOD connectors.
![ebaz4205_adapter](https://user-images.githubusercontent.com/1442215/110219236-b62da080-7ec6-11eb-8fe4-a9db97f4efef.png)

This way the adapter is kept simple and everyone would on use only the PMOD modules, they need.
You can use the available pmods or draw you own.
https://muselab-tech.aliexpress.com/store/5940159
https://store.digilentinc.com/boards-and-components/expansion-modules/pmods/

I added an extra place for OV7670 camera header. It would replace the two of the PMODS connectors.

For DVI/HDMI the differerential pairs are needed and they would be arranged to some connectors. The Idea is to use two PMOD connectors for either HDMI or VGA.
Here are some plans for HDMI, VGA and mandatory button and LED PMODs.
![pmod_pack](https://user-images.githubusercontent.com/1442215/110219252-d0677e80-7ec6-11eb-821a-a736b2e70028.png)

Are there any special pins on Data connectors  ? e.g. Should some pins to be avoided.

Any ideas of the design ?

**Comment by DavidJRichards** [comment]
HI, if you look at the ebaz board track layout you can see that the data i/o wires are mainly arranged in pairs, it would be best to carry this pairing through to the hdmi data connections to minimise cross-talk and ensure path lengths are as equal as possible for the hdmi data pairs. Perhaps you have done so already.
hth David.

**Reply by kasbert** [reply]
What is the acceptable difference in lengths ? There are now like 1590 vs. 1830 mils.
Should the wires be in tight pairs rather than spread evenly ? How close ?

--
Well, I suppose they don't charge by the length of the wire ;-), so I could make length to match to a couple of mils.

**Comment by kasbert** [comment]
Here's a link to the current version:
https://oshwlab.com/jarkko.sonninen/ebaz4205adap_copy

Note that this has not been produced yet and it is probably full of flaws.
